bin/autodetect/CCmdLineParser.cc

```diff
@@ -69,24 +68,14 @@ bool CCmdLineParser::parse(int argc,
                     "Optional field config file")
             ("modelplotconfig", boost::program_options::value<std::string>(),
                     "Optional model plot config file")
```

Comment:
Can `modelplotconfig`, `fieldconfig` and `limitconfig` also be removed now?

Also (nit) I just noticed the help text for `filtersconfig` and `eventsconfig` is not indented the same amount as all the other help text.